src/meta/src/hummock/hummock_manager.rs

```diff
@@ -858,11 +887,21 @@ where
         let versioning = versioning_guard.deref_mut();
         let mut current_version_id = VarTransaction::new(&mut versioning.current_version_id);
         let mut hummock_versions = VarTransaction::new(&mut versioning.hummock_versions);
+        let mut hummock_version_deltas =
+            VarTransaction::new(&mut versioning.hummock_version_deltas);
         let mut sstable_id_infos = VarTransaction::new(&mut versioning.sstable_id_infos);
         current_version_id.increase();
+        let mut new_version_delta = hummock_version_deltas.new_entry_txn_or_default(
+            current_version_id.id(),
+            HummockVersionDelta {
+                prev_id: old_version.id,
+                ..Default::default()
+            },
+        );
         let mut new_hummock_version =
```

Comment:
do we need to remove `VarTransaction` for `hummock_versions`

Reply:
> do we need to remove `VarTransaction` for `hummock_versions`

I wanted to remove it in the future

Reply:
ok